### PR TITLE
[improvement] Optimize PageInfo tool code

### PR DESCRIPTION
**Pull request by DaZuiZui**
Optimize paging tool code

## Verify this pull request

This pull request is code cleanup without any test coverage.

*(or)*

This pull request is already covered by existing tests, such as *(please describe tests)*.

(or)

This change added tests and can be verified as follows:

(or)

If your pull request contain incompatible change, you should also add it to `docs/docs/en/guide/upgrede/incompatible.md`

### CODE REVIEW

**Review by Gallardot — APPROVED** [review]
LGTM! Thanks.

**Review by SbloodyS — DISMISSED** [review]
+1

**Comment by SbloodyS** [comment]
Please run `mvn spotless:apply` to format code. @DaZuiZui

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java [hunk 1]
```diff
@@ -82,10 +82,8 @@ public Integer getTotalPage() {
         if (pageSize == null || pageSize == 0) {
             pageSize = 7;
         }
-        this.totalPage =
-                (this.total % this.pageSize) == 0
-                        ? ((this.total / this.pageSize) == 0 ? 1 : (this.total / this.pageSize))
-                        : (this.total / this.pageSize + 1);
+        
+        this.totalPage = Math.ceil(this.total / this.pageSize);
```
Comment (Gallardot) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java]:
```suggestion
        this.totalPage = (int)Math.ceil((double)this.total / (double)this.pageSize);
```

need to do some type conversion

**Review by Gallardot — CHANGES_REQUESTED** [review]

**Comment by DaZuiZui** [comment]
@SbloodyS Sorry, I can't use the mvn spotless:apply command. Can you help me?

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java [hunk 2]
```diff
@@ -82,10 +82,8 @@ public Integer getTotalPage() {
         if (pageSize == null || pageSize == 0) {
             pageSize = 7;
         }
-        this.totalPage =
-                (this.total % this.pageSize) == 0
-                        ? ((this.total / this.pageSize) == 0 ? 1 : (this.total / this.pageSize))
-                        : (this.total / this.pageSize + 1);
+        
+        this.totalPage = (int)Math.ceil((double)this.total / (double)this.pageSize);
```
Comment (Gallardot) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java]:
```suggestion
        this.totalPage = (int) Math.ceil((double) this.total / (double) this.pageSize);
```

 `mvn spotless:apply`

**Review by Gallardot — CHANGES_REQUESTED** [review]

**Comment by DaZuiZui** [comment]
@Gallardot @SbloodyS  thank

**Comment by SbloodyS** [comment]
> @SbloodyS Sorry, I can't use the mvn spotless:apply command. Can you help me?

You should run ti locally and push again.

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java
Comment (499636235) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/utils/PageInfo.java:1]: I prefer this:```this.totalPage = this.total / this.pageSize + 1;```

**Pull request closed without merging**